### PR TITLE
Update dropdown and placeholder

### PR DESCRIPTION
**Pull request by cal-smith**

### CODE REVIEW

File: src/utils/scroll.ts [hunk 1]
```diff
@@ -45,6 +46,16 @@ export const getScrollableParents = (node: HTMLElement) => {
...
 
+export const hasScrollableParents = (node: HTMLElement) => {
+	while (node.parentElement && node !== document.body) {
+		if (isScrollableElement(node)) {
+			return true;
+		}
+		node = node.parentElement;
+	}
+	return false;
+};
```
Comment (theiliad) [inline src/utils/scroll.ts:57]: 👍

File: src/placeholder/placeholder.service.ts [hunk 2]
```diff
@@ -10,59 +10,89 @@ import { Injectable } from "@angular/core";
...
+		component.destroy();
+		// const index = this.viewContainerRef.indexOf(component.hostView);
+		// if (index < 0) {
+		// 	return;
+		// }
 
-		// destroy the view
-		this.viewContainerRef.remove(index);
+		// // destroy the view
+		// this.viewContainerRef.remove(index);
```
Comment (theiliad) [inline src/placeholder/placeholder.service.ts:60]: ?

Comment (cal-smith) [reply src/placeholder/placeholder.service.ts:60]: Aw shoot, good catch ... definitely not supposed to be there

**Comment by cal-smith** [comment]
Hmm I thought I had reset my branch before making that other PR ... Oh well, nothing will be released either way ... @Donisius or @zvonimirfras definitely give this a review and let me know if there's anything to change - I'll address those in a new PR 😅 totally my bad

**Pull request merged**